src/main/kotlin/apply/infra/mail/AwsMailSender.kt

```diff
@@ -44,6 +50,32 @@ class AwsMailSender(
         client.sendEmail(request)
     }
 
+    override fun sendBcc(
+        toAddresses: Array<String>,
+        subject: String,
+        body: String,
+        files: Map<String, ByteArrayResource>
+    ) {
+        val applicationProperties = applicationProperties
+        val templateEngine = templateEngine
+        val multipartMimeMessage = message {
+            this.applicationProperties = applicationProperties
+            this.templateEngine = templateEngine
+            this.subject = subject
+            this.userName = mailProperties.username
+            this.recipient = Recipient(javaxMessage.RecipientType.BCC, toAddresses)
+            this.body = body
+            this.files = files
+        }.build()
+
+        val rawEmailRequest = multipartMimeMessage.getRawEmailRequest()
+        try {
+            client.sendRawEmail(rawEmailRequest)
+        } catch (ex: Exception) {
+            ex.printStackTrace()
+        }
+    }
+
     private fun createContent(data: String): Content {
         return Content(data).withCharset("UTF-8")
```

Comment:
```suggestion
        return Content(data).withCharset(Charsets.UTF_8.name())
```
코틀린에서 제공하고있는 상수를 활용해보면 어떨까요? 🤔